pkg/minikube/driver/driver.go

```diff
@@ -178,6 +180,11 @@ func IsVFKit(name string) bool {
 	return name == VFKit
 }
 
+// IsKrunkit checks if the driver is vfkit
```

Comment:
// IsKrunkit checks if the driver is vfkit
```suggestion
// IsKrunkit checks if the driver is krunkit
```